modules/ROOT/pages/tableofcontents.adoc

```diff
@@ -3,10 +3,15 @@
 :description: Insert a simple Table of Contents into the {productname} editor
 :keywords: tableofcontents, tableofcontents_depth, tableofcontents_class, tableofcontents_header, toc
 :pluginname: Table of Contents
+:pluginnamelower: table of contents
```

Comment:
Should this really be a variable? This is not a plugin name it's just plain text...

- [ ] Remove variable here

Reply:
I was thinking there were more cases but only found one, fixing now